packages/orbit-components/src/Table/TableCell/index.tsx

```diff
@@ -20,15 +20,16 @@ const whitespaceStyles: Record<WhiteSpace, string> = {
   "pre-wrap": "whitespace-pre-wrap",
 };
 
-const TableCell = ({
-  align = ALIGN_OPTIONS.LEFT,
-  scope,
-  as: Component = TYPE_AS.TD,
-  verticalAlign,
-  whiteSpace,
-  dataTest,
-  children,
-}: Props) => {
+const TableCell = (props: Props) => {
+  const {
+    align = ALIGN_OPTIONS.LEFT,
+    as: Component = TYPE_AS.TD,
+    verticalAlign,
+    whiteSpace,
+    dataTest,
+    children,
+  } = props;
```

Comment:
Why do we need this change?

Also, you're unpacking `as` but using `props.as` below.

Reply:
This is the only way I found

```tsx
const TableCell = ({
  align = ALIGN_OPTIONS.LEFT,
  as: Component = TYPE_AS.TD,
  verticalAlign,
  whiteSpace,
  dataTest,
  children,
  scope,
}: Props & { scope?: Scope }) => {
  return (
  ...
  ```
  
  but I don't like it, so we can keep your approach :+1: